sw/device/lib/crypto/drivers/rv_core_ibex.c

```diff
@@ -27,6 +34,33 @@ static void wait_rnd_valid(void) {
   }
 }
 
+hardened_bool_t ibex_disable_icache(void) {
```

Comment:
I think this makes sense. Requires the caller to create a var and handle the state in each function but avoids global var and state